tests/testthat/test_paramtest_blockForest_surv_blockforest.R

```diff
@@ -0,0 +1,32 @@
+test_that("surv.blockforest", {
+  learner = LearnerSurvBlockForest$new()
+  fun = blockForest::blockfor
```

Comment:
```suggestion
  fun = list(blockForest::blockfor, blockForest::blockForest)
```

Reply:
I think it's better not to add this, as there is a bunch of parameters that we need to exclude in the test and I don't think anyone will really use.